packages/react-dom/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -1189,6 +1256,39 @@ function pushStartTitle(
   return children;
 }
 
+function pushStartHead(
+  target: Array<Chunk | PrecomputedChunk>,
+  preamble: ?Array<Chunk | PrecomputedChunk>,
+  props: Object,
+  tag: string,
+  responseState: ResponseState,
+): ReactNodeList {
+  // Preamble type is nullable for feature off cases but is guaranteed when feature is on
+  target = enableFloat && isPreambleInsertion(tag) ? (preamble: any) : target;
```

Comment:
You don't need the `isPreambleInsertion()` checks. That was the point of creating the `pushStartHead` helper - now you know that it's a preamble insertion.

Reply:
brain broke